index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Dire's Portfolio</title>
+    <script src="https://kit.fontawesome.com/b7a7cffe76.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600&display=swap" rel="stylesheet" >
+</head>
+<body>
+  <main>
+      <div class="header">
+       <a href="">My Logo</a>
+      <nav><a href=""><i class="fas fa-bars"></i></a></nav>
+      </div> 
+    <section class="container">
+      <div class="intro">
+        <h1>Hi! I'M DIRE</h1> 
+          <h1>GLAD TO SEE <br>YOU!</h1>
+         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
```

Comment:
The font properties used here do not match the template to a certain level, this includes; font face, font weight, and font color

- [ ] Kindly update it accordingly, see reference below

Figma | Your copy 
------------ | ------------ 
![image](https://user-images.githubusercontent.com/67957533/133977070-bb8e375e-7771-4e23-8fce-c518e54b4be9.png) | ![image](https://user-images.githubusercontent.com/67957533/133977106-addea792-5a69-46ed-9fc7-2a47f609e289.png)